### PR TITLE
Add support for RTMPS

### PR DESCRIPTION
**Pull request by Qizot**

### CODE REVIEW

**Comment by Qizot** [comment]
I've added tests for RTMPS but the problem is how to keep the certificates to use them for testing? I would go for creating them in CI and then passing their paths via environmental variables but I have no idea how I can extend already existing CricleCI jobs to generate certs there and put them in certain directory so it is available for the next job.

File: lib/membrane_rtmp_plugin/rtmp/source/source.ex [hunk 1]
```diff
@@ -143,9 +190,16 @@ defmodule Membrane.RTMP.Source do
   end
 
   def handle_info(:start_receiving, _ctx, %{socket_retries: retries} = state) do
-    case :gen_tcp.controlling_process(state.socket, state.receiver_pid) do
+    module = if state.use_ssl?, do: :ssl, else: :gen_tcp
+
+    case module.controlling_process(state.socket, state.receiver_pid) do
       :ok ->
-        :ok = :inet.setopts(state.socket, active: :once)
+        if state.use_ssl? do
```
Comment (shuntrho) [inline lib/membrane_rtmp_plugin/rtmp/source/source.ex:197]: Why not use `module.setops(...)` here?

File: lib/membrane_rtmp_plugin/rtmp/source/message_handler.ex [hunk 2]
```diff
@@ -293,4 +297,8 @@ defmodule Membrane.RTMP.MessageHandler do
         parse_packet_messages(<<>>, message_parser, [{nil, step} | messages])
     end
   end
+
+  @compile {:inline, socket_module: 1}
```
Comment (shuntrho) [inline lib/membrane_rtmp_plugin/rtmp/source/message_handler.ex:301]: I wonder if we really need this little bit of performance here, I leave this decision up to you

File: test/test_helper.exs [hunk 3]
```diff
@@ -1 +1 @@
-ExUnit.start(capture_log: true)
+ExUnit.start(capture_log: true, exclude: [:rtmps])
```
Comment (shuntrho) [inline test/test_helper.exs:1]: Why do we exclude `rtmps` here?

**Review by shuntrho — CHANGES_REQUESTED** [review]

File: test/test_helper.exs (hunk 3 above)
Comment (Qizot) [reply test/test_helper.exs:1]: Read the comment I left above.

File: lib/membrane_rtmp_plugin/rtmp/source/source.ex (hunk 1 above)
Comment (Qizot) [reply lib/membrane_rtmp_plugin/rtmp/source/source.ex:197]: because `:ssl` module doesn't work with that

**Review by shuntrho — APPROVED** [review]

**Pull request merged**